connectors/src/api_server.ts

```diff
@@ -84,6 +88,10 @@ export function startServer(port: number) {
   );
 
   app.post("/webhooks/:webhook_secret/slack", webhookSlackAPIHandler);
+  app.post(
+    "/webhooks/:webhook_secret/slack_reactions",
```

Comment:
NIT: `/webhooks/:webhook_secret/slack_reactions` to `/webhooks/:webhook_secret/slack_message_interaction`? Does that make sense?